### DISCUSSION TITLE
Will moving the destination folder break --update if the db is in a 3rd location?

### DISCUSSION THREAD

**Discussion by swduncan** [RhetTbull/osxphotos#988] [Q&A] [answered]
I'm exporting to director a, and have the db in directory b, and I'd like to move the destination direction from a to c. If I do so, will --update recognize what's already there, or will it see directory c as empty and start exporting from scratch?

**Comment by RhetTbull** [comment, marked as answer]
Yes, you can move the destination directory *as long as you also move the .osxphotos_export.db file that is in the export directory*.  The `.osxphotos_export.db` file contains all the data osxphotos needs to continue the export and it will work fine if you move the top level export directory (all paths are stored relative to the export location by osxphotos).

**Reply by swduncan** [reply]
So if all paths are stored relative to the export location, then if the export db file is in a third location (my local ssd) then it should still work, yes?

**Reply by RhetTbull** [reply]
Yes. The export code re-assembles the paths based on the export destination (not the location of the export db) and the relative path in the database.

**Reply by swduncan** [reply]
Perfect! Thanks!

**Reply by RhetTbull** [reply]
Sorry -- I misunderstood the original question (assumed you meant "Photos DB" when you said "db").  If you hadn't set a custom export db location, then you'd have to move the export db (.osxphotos_export.db) but in your case, you can leave the export database where it is.
